js/apps/admin-ui/src/realm-settings/security-defences/BruteForceDetection.tsx

```diff
@@ -127,8 +138,42 @@ export const BruteForceDetection = ({
                 )}
               />
             </FormGroup>
+            {permanentLockout && (
```

Comment:
I'm still a bit confused about the `max temporary lockouts` being shown when you have permanent lockout enabled. Isn't better to just hide it and force the value to `0` ?

Reply:
Yeah, my bad. I re-read the UI and remember our last discussion.